### PR TITLE
ENH: New table to facilitate matching  `chain_id -> chain_name` in prices tables

### PR DESCRIPTION
**Pull request by cc7768**
Brief comments on the purpose of your changes:

Many events on-chain (especially bridge events) track chains by their chain id rather than the identifier used in the `prices.usd` table. This PR adds a new table to the `chain_info` schema called `dune_chains` which is a simple mapping from `chain_id` to `dune_name` where `dune_name` corresponds to the naming conventions in the prices table.

We are using a version of this internally and thought it would be of more general use.

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @cc7768 

chain_id checked in https://chainlist.org/ except id for `solana`

**Comment by cc7768** [comment]
Cheers! Thanks for the review.

That Solana chain id came up when I googled but I'm not sure it's a thing tbh. Maybe we should remove it unless we have a more reputable source than "I googled it and saw a single result" :sweat_smile: 

Thoughts? I'll open a suggestion with that change and you can accept it if you agree.

File: models/chain_info/chain_info_dune_chains.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  VALUES
+    ('ethereum', 1),
+    ('optimism', 10),
+    ('bnb', 56),
+    ('gnosis', 100),
+    ('polygon', 137),
+    ('fantom', 250),
+    ('arbitrum', 42161),
+    ('avalanche_c', 43114),
+    ('solana', 1399811149)
```
Comment (cc7768) [inline models/chain_info/chain_info_dune_chains.sql:22]:
```suggestion
```

Comment (Hosuke) [reply models/chain_info/chain_info_dune_chains.sql:22]: We can add back solana id when we have more confident source.

**Review by jeff-dude — APPROVED** [review]
works for me -- thank you for the contribution and making general purpose!

**Pull request merged**